src/foundry/common/prosemirror/content-link-plugin.d.mts

```diff
@@ -42,3 +34,14 @@ declare class ProseMirrorContentLinkPlugin extends ProseMirrorPlugin {
    */
   protected _onDrop(view: EditorView, event: DragEvent, slice: Slice, moved: boolean): void | true;
 }
+
+declare namespace ProseMirrorContentLinkPlugin {
+  interface ProseMirrorContentLinkOptions {
+    /** The parent document housing this editor. */
+    document?: foundry.abstract.Document<any, any, any> | undefined;
```

Comment:
Nit; you should be able to say `foundry.abstract.Document.Any`. I'll always encourage this because it lets us more easily add generic parameters etc.